src/Microsoft.Extensions.Configuration.AzureAppConfiguration/Microsoft.Extensions.Configuration.AzureAppConfiguration.csproj

```diff
@@ -17,17 +17,11 @@
     <PackageReference Include="Azure.Data.AppConfiguration" Version="1.3.0-beta.3" />
     <PackageReference Include="Azure.Messaging.EventGrid" Version="4.7.0" />
     <PackageReference Include="Azure.Security.KeyVault.Secrets" Version="4.3.0" />
-    <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="3.1.18" />
-    <PackageReference Include="Microsoft.Extensions.Logging" Version="3.1.18" />
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="6.0.0" />
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="6.0.0" />
     <PackageReference Include="System.Text.Json" Version="4.7.2" />
-  </ItemGroup>
-
-  <ItemGroup Condition="'$(TargetFramework)' == 'net7.0'">
-	<PackageReference Include="Microsoft.Extensions.Configuration" Version="7.0.0" />
-  </ItemGroup>
-
-  <ItemGroup Condition="'$(TargetFramework)' != 'net7.0'">
-	<PackageReference Include="Microsoft.Extensions.Configuration" Version="3.1.18" />
+    <PackageReference Include="Microsoft.Extensions.Configuration" Version="6.0.1" />
+    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="6.0.0" />
```

Comment:
Not sure if I've added the correct version for `Microsoft.Bcl.AsyncInterfaces`, but the examples threw an error saying version 6.0.0 is missing until I added it. The examples/tests all compile and run now, with net48 (for console apps), net6.0 and net7.0 as target frameworks.

Reply:
I don't see it listed as a dependency before. HOw did this work?

Reply:
Microsoft.Bcl.AsyncInterfaces is implicitly loaded with Microsoft.Extensions.DependencyInjection.Abstractions and Microsoft.Extensions.Logging packages. I'm not sure if this explicit reference is needed. 
Which example is broken if we remove this reference? Maybe we need to update some reference in the example itself.

Reply:
I tried running the ConfigStoreDemo and it failed with that error, I don't think there's anything to update there other than the DotNetCliToolReference.

Reply:
Also worth mentioning, if I don't add it there's a warning for all of the tests too that says it can't resolve the version of Microsoft.Bcl.AsyncInterfaces.

Reply:
Turns out the issue is that the packages we upgraded, Microsoft.Extensions.Logging,  and Microsoft.Extensions.DependencyInjection.Abstractions, both have dependencies on version 6.0.0 and later of bcl.asyncinterfaces. Version 3.1.18 of these same packages had no dependencies on bcl.asyncinterfaces, which is why we just now get this error. The solution should be to include version 6.0.0 as it is in this PR.

Reply:
> The conflict is because something is referencing 1.0.0

That's the AppConfig SDK. We need to add a new explicit reference to override their version.